server/app/views/style/ReferenceClasses.java

```diff
@@ -138,6 +138,8 @@ public final class ReferenceClasses {
   public static final String BT_DATE = "cf-bt-date";
   public static final String BT_APPLICATION_ID = "cf-application-id";
   public static final String BT_EMAIL = "cf-bt-email";
+  public static final String BT_API_KEY_ID = "cf-api-key-id";
```

Comment:
These should have `bt` in them for Browser Tests

Reply:
Thanks. Done.